app/utils/google_drive_api.py

```diff
@@ -47,13 +49,11 @@ def get_file_in_google_drive(google_api_call, file_to_upload):
                                         fields="id").execute()
```

Comment:
NIT: @Johan-Cuevas, can you place lines 41-49 under and `else` block to decrease reading confusion for future :) 

also fyi `nit` is review-jargon that means "this is, a nit-picky small change that may not be very important, but is technically correct."